packages/squiggle-lang/__tests__/TS/SampleSet_test.ts

```diff
@@ -0,0 +1,175 @@
+import { Distribution } from "../../src/js/index";
+import { expectErrorToBeBounded, failDefault } from "./TestHelpers";
+import * as fc from "fast-check";
+
+// Beware: float64Array makes it appear in an infinite loop.
+let arrayGen = () =>
+  fc.float32Array({
+    minLength: 10,
+    maxLength: 10000,
+    noDefaultInfinity: true,
+    noNaN: true,
+  });
+
+describe("SampleSet: cdf", () => {
+  let n = 10000;
+  test("at the highest number in the distribution is within epsilon of 1", () => {
```

Comment:
can we say ``is close to 1`` instead of ``epsilon``? This is already a common description/notation in Jest.

Reply:
roger.